### ISSUE TITLE
Feature Request - Directory Listing to the Side of the PDF being viewed

### ISSUE THREAD

**Issue by rli77** [sumatrapdfreader/sumatrapdf#916]
**I was searching for something to meet the following requirements-**
- lightweight PDF viewer
- no need for advanced options
- [MAIN] the ability to browse through PDFs in the selected folder in alphabetical order by keyboard

SumatraPDF has provided me with the above but not the below :o)

**More Detailed Requirments**
I would love for it to allow me to select a directory rather than a single file.
Show a list of all the PDF (and Word) files in that directory
Let me use the keyboard to next-page > next-page > next-page and then at end open the next file

**Comment by febkor** [comment]
I'm not sure I understand exactly what you mean.
Sumatra does offer you the ability to open the prev/next PDF in the folder (in alphabetical order) by using ctrl+shift+left/right arrow. 

See the [manual](https://www.sumatrapdfreader.org/manual.html) for more features.

**Comment by rli77** [comment]
Hello. Indeed you are right. I would find it useful to see the files in the
chosen directory listed. As I read though the pages I would like to see the
particular PDF highlighted in the list. The when it moves onto the next PDF
the highlight moves in the list. Maybe a screen button for page turns or
simpler key combination.
I can do a mock-up if that helps.
Thanks

**Comment by GitHubRulesOK** [comment]
@kjk the requested function is basically what apps like Dopus or File Commanders do
They use Word or SumatraPDF as document format viewers thus the OP request is best served by Windows Folder handlers with formathandlers like Universal Viewer etc that use Slister or similar.

<img width="1314" height="851" alt="Image" src="https://github.com/user-attachments/assets/dfb0b3e3-bf08-4318-8fd6-978e0523ebc6" />

**Comment by GitHubRulesOK** [comment]
@kjk the current navigator addresses most if not all of this request

<img width="782" height="668" alt="Image" src="https://github.com/user-attachments/assets/77f008d6-ed7d-40e2-949c-6a055a5d4cea" />

**Comment by kjk** [comment]
Yes, this is implemented as `Ctrl + K`, `Navigate Files In Folder`
